easybuild/easyconfigs/a/ARAGORN/ARAGORN-1.2.41-foss-2020b.eb

```diff
@@ -0,0 +1,43 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# Updated: Denis Kristak
+
+easyblock = 'CmdCp'
+
+name = 'ARAGORN'
+version = '1.2.41'
+
+homepage = 'http://mbio-serv2.mbioekol.lu.se/ARAGORN/'
+description = "http://www.ansikte.se/ARAGORN/"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['http://www.ansikte.se/ARAGORN/Downloads/']
+sources = [
+    {'filename': '%(namelower)s%(version)s.c', 'extract_cmd': 'cp %s aragorn.c'},
+    {'filename': 'aragorn.1', 'extract_cmd': 'cp %s aragorn.1'}  # man page
+]
+checksums = [
+    '92a31cc5c0b0ad16d4d7b01991989b775f07d2815df135fe6e3eab88f5e97f4a',  # aragorn1.2.41.c
+    '6e30f6d9da644197064744f033176969e77855964ed8d50ea00c38a3aea2c92e',  # aragorn.1
+]
+
+cmds_map = [
+    (".*", "$CC $CFLAGS -o aragorn aragorn.c"),  # TODO since there are 2 sources, install command repeats twice
+]
+
+files_to_copy = [
+    (['aragorn'], 'bin'),
+    (['aragorn.1'], 'share/man/man1'),
+]
```

Comment:
Maybe this is better (to avoid the duplicate compile)?

```python
cmds_map = [
    (".*.c", "$CC $CFLAGS -o aragorn aragorn.c"),
    (".*.1", "mkdir -p share/man/man1 && cp aragorn.1 share/man/man1"),
]

files_to_copy = [
    (['aragorn'], 'bin'),
    'share',
]
```

Reply:
done